src/components/credits-footer.js

```diff
@@ -13,7 +14,7 @@ function renderUser(user) {
 
 function renderDocumentContributors(doc, t) {
   const cons = doc.contributors.map((user, index) => (
-    <Fragment key={index.toString()}>
+    <Fragment key={user.email}>
```

Comment:
Isn't `doc.contributors` an array of string? 🤔

Reply:
Got it.  You'll need to change the key to `user._id` then, cause email is not unique in contributors now since we can have internal and externally imported users. I got a non unique key error in the console.